discovery-provider/compose/docker-compose.web-server.yml

```diff
@@ -7,19 +7,25 @@ services:
   web-server:
     build: ../.
     restart: always
-    command: sh -c '/wait && ls && exec ./scripts/dev-server.sh'
     ports:
-      - '5000:5000'
+      - '${audius_server_port}:${audius_server_port}'
     env_file:
       - .env
     environment:
-      - WAIT_HOSTS=redis-server:6379,ipfs-node:5001
       - audius_db_run_migrations=false
+      - FLASK_RUN_PORT=${audius_server_port}
+      - audius_redis_url=redis://${COMPOSE_PROJECT_NAME}_redis-server_1:6379/00
+      - audius_ipfs_host=${COMPOSE_PROJECT_NAME}-ipfs-node
+      - dev=true
```

Comment:
can we rename this to audius_discprov_dev_mode or something? i want to use this for some other stuff

Reply:
the way we access it inside the discovery config is parse out the env vars with audius_ prefix